### DISCUSSION TITLE
is it safe to use milvus deployed with Ansible in production environment?  What nodes should I deploy to distribute RAM consumption across different servers?

### DISCUSSION THREAD

**Discussion by martinenkoEduard** [milvus-io/milvus#22894] [Q&A and General discussion] [answered]
is it safe to use milvus deployed with Ansible in production environment?  What nodes should I deploy to distribute RAM consumption across different servers?

**Comment by yhmo** [comment, marked as answer]
Ansible can only be used in test environments. It is not recommended that you deploy Milvus distributed clusters in this way in production environments.
